packages/vue/src/DataDisplay/ListDetail/OcListDetail.vue

```diff
@@ -34,7 +38,7 @@ const isVertical = computed(() => props.alignment === "vertical");
     >
       {{ label }}
 
-      <CopyTooltip :value="content">
+      <CopyTooltip v-if="showCopyTooltip" :value="content">
```

Comment:
These are the actual changes. The rest are formatting changing from `npm run format`